charts/prometheus-blackbox-exporter/values.yaml

```diff
@@ -156,3 +156,8 @@ networkPolicy:
   enabled: false
   # Limit access only from monitoring namespace
   allowMonitoringNamespace: false
+
+## dnsPolicy and dnsConfig for Deployments and Daemonsets if you want non-default settings.
+## These will be passed directly to the PodSpec of same.
+# dnsPolicy:
+# dnsConfig:
```

Comment:
I suggest to remove the comment "#". If the values key exits then some IDEs can prove auto-completion for it.

Reply:
Changed and resolved in https://github.com/prometheus-community/helm-charts/pull/42/commits/27e8ee1f9e4ce2320e3a29f55f3e052a95da5075